### ISSUE TITLE
Random results with 0.0 confidence on demo.py

### ISSUE THREAD

**Issue by ashkanmradi** [clovaai/deep-text-recognition-benchmark#370]
I re-trained a **_TPS-ResNet-BiLSTM-Attn_** model to predict the results for Persian numbers and dates. In other words, my character set contains 0-9 (Persian characters) plus a slash ("/") character.
 
When I use the **_test.py_** script to check the model accuracy on unseen data, I get a good result (about 96.3%).

However, when I use the **_demo.py_** script, I get random results for all input images. 

![image](https://user-images.githubusercontent.com/15668878/202277859-6805b936-7e42-43e0-a626-b414c6a616b2.png)

I would appreciate it if you could help me with my problem.

**Comment by krammnic** [comment]
Same problem

**Comment by krammnic** [comment]
Tried CRNN model, same result

**Comment by krammnic** [comment]
@ashkanmradi I fixed it with reinstalling all libraries and this repo.

**Comment by ashkanmradi** [comment]
@krammnic Thanks for your solution. 
Just one question. Did you train a new model after _reinstalling all libraries and this repo_, or you load your previously learnt model?

**Comment by krammnic** [comment]
Yes, I trained new model, old one returned random results with 0.0 confidence.

**Comment by Al-Moatasem** [comment]
Hello @krammnic
Can you share with us the requirements.txt

**Comment by krammnic** [comment]
> 

I learned a new model
